src/promptflow-devkit/promptflow/_sdk/entities/_flows/flex.py

```diff
@@ -45,6 +45,10 @@ def language(self) -> str:
     def additional_includes(self) -> list:
         return self._data.get("additional_includes", [])
 
+    @property
```

Comment:
Also add in flex flow's schema?

Reply:
![image](https://github.com/microsoft/promptflow/assets/17938940/2dfc036a-a0f0-4b6c-9d3e-67bdde3fa0cd)
it has been added in #2622